src/Microsoft.Sbom.Api/Executors/PackageInfoJsonWriter.cs

```diff
@@ -67,21 +67,22 @@ internal async Task GenerateJson(
                 var generationResult =
                     manifestGeneratorProvider.Get(sbomConfig.ManifestInfo).GenerateJsonDocument(packageInfo);
 
-                var recordedAnyDependencies = false;
+                //var recordedAnyDependencies = false;
```

Comment:
I'm assuming we're commenting this out instead of deleting it because we imagine we might want to reenable some version of this behavior some day? If that's the case, can we add a comment above this section with a quick statement about that.